packages/rest/src/validation/request-body.validator.ts

```diff
@@ -7,18 +7,25 @@ import {
   RequestBodyObject,
   SchemaObject,
   SchemasObject,
-  isReferenceObject,
 } from '@loopback/openapi-v3-types';
 import * as AJV from 'ajv';
 import * as debugModule from 'debug';
 import * as util from 'util';
 import {HttpErrors} from '..';
-import {RestHttpErrors} from '../coercion/rest-http-error';
+import {RestHttpErrors} from '..';
+import {AnyObject} from '@loopback/repository';
 
 const toJsonSchema = require('openapi-schema-to-json-schema');
-
 const debug = debugModule('loopback:rest:validation');
 
+/**
+ * Validate the body from a request fits the JSON schema.
```

Comment:
I am proposing a slightly different phrasing. Please note that the OpenAPI schema of a request body can be provided in many different ways, e.g. via `app.api()`, `@api()` decorator on a controller or `app.route()` method.

```
Check whether the request body is valid according to the provided OpenAPI schema.
The JSON schema is generated from the OpenAPI schema which is typically defined 
by `@requestBody()`.

The function throws when the data object is not valid.
```

Reply:
true...I overlooked the api first approach.